doc/production-solr-collection-swap-process.md

```diff
@@ -1,42 +1,43 @@
 
 # Production Solr Collection Swap Process Steps
 
-This document outlines the expected steps needed to complete the Full Reindex Process to go from exporting data from Alma to having it live in the Production Library Search. This is intended as a checklist for the person responsible for shepherding the process to ensure that the required manual steps are followed.
+This document outlines the expected steps needed to complete the catalog full reindex process from exporting data from Alma to deployment on the production Library Search site. This is intended as a checklist for the person responsible for shepherding the process to ensure that the required manual steps are followed.
 
 These steps are broken down into three sections as these parts can be run sequentially, but they may also be run independently.
 
 ## Full Reindex
 
-1. Initiate Alma FTP Export and wait for completion.
+1. Initiate appropriate Alma SFTP exports via pubishing profiles in Alma and wait for completion.
 1. Trigger the `catalog_move_alma_sftp_to_s3` Dag and wait for completion.
    1. Skip this step if you want to reuse the s3 alma_sftp files from the last run.
 1. Trigger the `boundwith_move_alma_sftp_to_s3` Dag and wait for completion.
    1. Skip this step if you want to reuse the s3 alma_sftp files from the last run.
 1. Turn off the catalog_preproduction_oai_harvest
-1. Set variables required for the full reindex:
-   1. Ensure the variable `CATALOG_PRE_PRODUCTION_SOLR_CONFIG` is using the correct version of solr configs
-   1. Ensure the variable `PRE_PRODUCTION_COB_INDEX_VERSION` is the correct version of cob index
+1. Set the Airflow variables required for the full reindex:
+   1. Ensure the variable `CATALOG_PRE_PRODUCTION_SOLR_CONFIG` is using the correct version of solr configs. See the [tul_cob-catalog-solr releases](https://github.com/tulibraries/tul_cob-catalog-solr/releases) to confirm latest version, if needed. 
+   1. Ensure the variable `PRE_PRODUCTION_COB_INDEX_VERSION` is the correct version of cob_index. See the  [cob_index releases](https://github.com/tulibraries/cob_index/releases) to confirm latest version, if needed. 
    1. Update the `CATALOG_PRE_PRODUCTION_SOLR_COLLECTION` variable to be None.
-1. Trigger `catalog_full_reindex` Dag and wait for completion
+1. Trigger `catalog_full_reindex` Dag and wait for completion. 
+1. Once a new preproduction Solr collection is created via `catalog_full_reindex`, turn on the `catalog_preproduction_oai_harvest` Dag. 
 
 ### Caveats and Incidentals
-* `cob_datapipeline` deployments to airflow-prod only happen on new releases for that repo (plus verified CI deployment step).
-* `ansible-playbook-airflow` prod updates need shepherding from qa to main branch and then a verified deployment step in the CI.
```

Comment:
@ebtoner I'm not sure if there is a better way to say it, but I think it is helpful for people to know that there is a verification step.  Releases don't trigger automatically, someone has to go to circleCI and approve it in order for changes to deploy.

Reply:
I've reverted these changes.

Reply:
Sorry I got mixed up about what is on linode vs. k8s!